components/TreasuryAccount/AccountItem.tsx

```diff
@@ -18,42 +24,129 @@ const AccountItem = ({
     displayPrice,
     isSol,
   } = getTreasuryAccountItemInfo(governedAccountTokenAccount, governanceNfts)
+
   const {
     setCurrentCompactView,
     setCurrentCompactAccount,
   } = useTreasuryAccountStore()
+
   const connection = useWalletStore((s) => s.connection)
+
   async function handleGoToAccountOverview() {
     setCurrentCompactView(ViewState.AccountView)
     setCurrentCompactAccount(governedAccountTokenAccount, connection)
   }
 
-  return (
+  // Target the appropriate pubkey depending if the account is SOL native (wallet) or a Token Account
+  const pubkey = isSol
+    ? governedAccountTokenAccount.transferAddress
+      ? governedAccountTokenAccount.transferAddress
+      : undefined
+    : governedAccountTokenAccount.governance?.pubkey
+
+  const { ownedTokenAccountsInfo } = useGovernanceUnderlyingTokenAccounts(
+    pubkey
+  )
+
+  const treasuryOwnerAccountsRef = createRef<HTMLAnchorElement>()
+
```

Comment:
```suggestion
if (!pubkey) return null;
```

Reply:
If pubkey is equal to null I still want to display the component